Dockerfile

```diff
@@ -13,25 +13,17 @@ RUN cargo build --locked --profile ${PROFILE}
 # RUN cargo test --release --all
 
 # This is the 2nd stage: a very small image where we copy the Polkadot binary."
-FROM docker.io/library/ubuntu:20.04
+FROM gcr.io/distroless/cc-debian11@sha256:c2e1b5b0c64e3a44638e79246130d480ff09645d543d27e82ffd46a6e78a3ce3
```

Comment:
https://console.cloud.google.com/gcr/images/distroless/global/cc-debian11@sha256:3516ad5504f54aeaea0444dfd5044cc5969653e49b594cecb47dc400ea6f6820/details - seems to be the latest